OrbitBase/include/OrbitBase/Action.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORIBIT_BASE_ACTION_H_
+#define ORIBIT_BASE_ACTION_H_
+
+// Actions are executed by MainThreadExecutor
+// The Action is an abstract class which can
+// be executed. See below for basic implementations for
+// this class: VoidAction and UnaryAction.
```

Comment:
Where are they?

Reply:
I deleted them after simplifying things - fixed the comment.